### ISSUE TITLE
randit bug

### ISSUE THREAD

**Issue by BruceSherwood** [kovidgoyal/rapydscript-ng#185]
I don't know whether you still fix bugs, but here is a minor bug that was reported to me. The following program never displays -2 :
from random import randint
r = 0
while r < 20:
    print(randint(-2,2))
    r += 1

**Comment by kovidgoyal** [comment]
randint is defined as

```
def randint(a, b):
    return int(random()*(b-a+1) + a)

```

the only way that can not return -2 is if the JS random() function never
returns zero. The documentation for the Math.random() function states it
returns a number in [0, 1).
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random

So this would be a bug in the underlying JS engines' Math.random() function

**Comment by kovidgoyal** [comment]
Actually, possibly that module is not using Math.random(). Let me look at it in detail when I have a minute.

**Comment by kovidgoyal** [comment]
Yeah looks like it uses a custom random() function based on RC4 that has extremely low probability of generating 8 consecutive zero bytes. You'd have to ask @atsepkov who wrote that function why that is.
